configure.ac

```diff
@@ -60,7 +60,7 @@ AC_TYPE_LONG_LONG_INT
 
 # Checks for library functions.
 AC_FUNC_FORK
-AC_CHECK_FUNCS([gettimeofday memset strstr])
+AC_CHECK_FUNCS([gettimeofday memset strstr strdup])
```

Comment:
It depends the level of coverage you want. 
strdup and strndup are both available when the user doesn't mess up with feature_test_macros and restrict itself only with the -std=gnuXX flag to enable gnu extensions. In the worst case the user namespace may end up with one extra symbol in these very specific cases. 
The reason I did not check for both strdup and strndup on configure is to avoid extra macros in code.

What do you recommend me to approach this?